server/src/test/java/io/spine/server/stand/given/StandTestEnv.java

```diff
@@ -123,6 +124,14 @@ public static Stand newStand(boolean multitenant, Repository<?, ?>... repositori
         return context.stand();
     }
 
+    public static Stand newStand(CommandDispatcher... dispatchers) {
+        BoundedContextBuilder builder = BoundedContextBuilder.assumingTests();
+        Arrays.stream(dispatchers)
+              .forEach(builder::addCommandDispatcher);
```

Comment:
Let's use a loop instead.

I'd accept this if there was at least one intermediate operation (e.g. `map`, `flatMap`, `filter`).

Reply:
Also, this should've been a `reduce()` or maybe a `collect()`, not a `forEach()`